benchmarks/src/main/scala/zio/TestRuntimeBenchmark.scala

```diff
@@ -0,0 +1,40 @@
+package zio
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+import zio.IOBenchmarks._
+import zio.test.TestRuntime
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+class TestRuntimeBenchmark {
```

Comment:
How do the benchmarks look at this point?

Reply:
I added new benchmark results below.